src/main/webapp/index_script.js

```diff
@@ -13,30 +13,22 @@
 // limitations under the License.
 
 document.addEventListener("DOMContentLoaded", async function() {
-  let organizations = [{
-    "name": "Organization A",
-  },
-  {
-    "name": "Organization B",
-  },
-  {
-    "name": "Organization C",
-  },
-  {
-    "name": "Organization D",
-  },
-  {
-    "name": "Organization E",
-  },
-  {
-    "name": "Organization F"
-  }];
   const organizationArea = document.getElementById('organization-list');
-  organizations.forEach((organization) => {
-    organizationArea.appendChild(createOrganization(organization));
-  });
+  renderOrganizations(organizationArea);
```

Comment:
instead of a function, can you create a class that
  knows how to fetch the data, storing it in a local variable
  knows how to render the stored data when it arrives?

Reply:
I adjusted this branch to integrate the code sarah wrote for the popups, and that code stores the organization list in a local variable after fetching it and then renders it once it arrives.